sound/soc/sof/intel/hda-loader.c

```diff
@@ -174,7 +174,7 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, int stream_tag)
 			__func__);
 
 err:
-	hda_dsp_dump(sdev, SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX);
+	snd_sof_dsp_dbg_dump(sdev, SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX);
```

Comment:
Commit message is cryptic, what can be extended exactly?

Reply:
im going to drop this commit for now but it is going to be needed at some point and I will revisit it then